app/webpack/lifelists/show/components/details_view.jsx

```diff
@@ -145,7 +145,7 @@ const DetailsView = ( {
       <div className="stats">
         <span className="stat">
           <span className="attr">
-            { I18n.t( "views.lifelists.observed_species" ) }
+            { I18n.t( "views.lifelists.observed_rank", { rank: rankLabel( lifelist.speciesViewRankFilter ) } ) }
```

Comment:
One thing here that's a little confusing - we use the term "Leaves" in the dropdown menu, and this is the default value for taxa filtering, but we'd like to use the term "Species" in this label when filtering by leaves or species. This is a little weird, and technically wrong, but the term "leaves" is confusing for people and we're already conflating the two terms in other places. Since "Leaves" is the default, we'd rather the first thing people see is "Observed species". So if the rank chosen is either "leaves" or "species" the label should read "Observed species".

It's also a little strange to me when "Children" is selected with there's no details taxon to see "Observed children" with no context. But that's probably outside the scope of this ticket. I think the best thing to do is not offer "Children" as a selectable value if there's no details taxon, or get rid of "children" entirely - not sure if anyone uses it

Reply:
I think I understand @pleary. Originally I thought "leaves" were shorthand for subsp., variety, cultivar, etc. It seems like it's more like terminal nodes for the selected taxon? Operating on that assumption I think the latest commit should address it.

Reply:
Yes leaves are terminal nodes in the tree. We go into a little [more detail here](https://www.inaturalist.org/pages/how_inaturalist_counts_taxa) if you're interested. Thanks - this change is looking great and is updating the labels as request